nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestCommentNotification.java

```diff
@@ -66,8 +70,12 @@
 @Features({ NotificationCommentFeature.class, SmtpMailServerFeature.class })
 public abstract class AbstractTestCommentNotification {
 
+    protected static final String ADMINISTRATOR = "Administrator";
+
+    protected static final String ANY_COMMENT_MESSAGE = "any Comment message";
+
     @Inject
-    protected NotificationService notificationService;
+    protected NotificationManager notificationManager;
```

Comment:
why changing it from `Service` to `Manager`

Reply:
NotificationService is an implementation of NotificationManager. We usually use the interfaces